src/test/java/com/ays/assignment/controller/AssignmentSystemTest.java

```diff
@@ -94,6 +97,57 @@ void givenValidAssignmentSaveRequest_whenUserUnauthorizedForSaving_thenReturnAcc
 
     }
 
+    @Test
+    void givenNothing_whenUserAssignmentFound_thenReturnAssignmentUserResponse() throws Exception {
+
+        // When
+        Assignment mockAssignment = new AssignmentBuilder()
+                .build();
+
+        // Then
+        String endpoint = BASE_PATH.concat("/assignment");
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .get(endpoint, userTokenFive.getAccessToken());
+
+        AssignmentUserResponse mockAssignmentUserResponse = ASSIGNMENT_TO_ASSIGNMENT_USER_RESPONSE_MAPPER.map(mockAssignment);
+        AysResponse<AssignmentUserResponse> mockAysResponse = AysResponse.successOf(mockAssignmentUserResponse);
+        mockMvc.perform(mockHttpServletRequestBuilder)
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(mockAysResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .exists());
+
+    }
+
+
+    @Test
+    void givenNothing_whenUnauthorizedForGettingUserAssignment_thenReturnAccessDeniedException() throws Exception {
```

Comment:
`givenNothing_` yerine hiç yazmayabiliriz doğrudan `when...` ile başlar

Reply:
İlgili test class'ındaki method isimlerini düzelttim. İsimleri değiştirince test order değiştiği için verileri farlı sırada değiştiriyordu. Bu yüzden testlerde veriden kaynaklı hata oluşuyordu. `@Order` anotasyonu ile doğru sırada veriyi güncellemelerini sağladım.